index.js

```diff
@@ -71,10 +80,23 @@ function getName(/* code here */) {
  *         and returns a string like `Hello, my name is {name}`.
  *         where `{name}` is the name passed into `makeSmartPerson`.
 */
-function makeSmartPerson(/* code here */) {
-  /* code here */
+function makeSmartPerson(name) {
+  const obj = {
+    name: name,
+    sum: function(a, b){
+      return a + b;
+    },
+    speak: function() {
+      return `Hello, my name is ${this.name}` ;
+    }
+  };
+  return obj;
 }
 
+var smartPerson = makeSmartPerson('Luke')
+
+smartPerson.speak()
+
 /**
```

Comment:
lines 96 and 98: The tests already have information to test your function, you don't have to do this.